directory/directory_test.go

```diff
@@ -31,10 +31,14 @@ func TestGetPutManifest(t *testing.T) {
 	defer os.RemoveAll(tmpDir)
 
 	man := []byte("test-manifest")
+	md, err := manifest.Digest(man)
+	require.NoError(t, err)
 	dest, err := ref.NewImageDestination(nil)
 	require.NoError(t, err)
 	defer dest.Close()
-	err = dest.PutManifest(man)
+	err = dest.PutManifest(man, &md)
+	assert.NoError(t, err)
+	err = dest.PutManifest(man, nil)
```

Comment:
The two written manifests should be different…

Reply:
Ah, right.  Fixed.